bears/typescript/TSLintBear.py

```diff
@@ -0,0 +1,48 @@
+import json
+
+from coalib.bearlib.abstractions.Lint import Lint
+from coalib.bears.LocalBear import LocalBear
+from coalib.misc.Shell import escape_path_argument
+from coalib.results.Result import Result
+
+
+class TSLintBear(LocalBear, Lint):
+    executable = 'tslint'
+
+    def process_output(self, output, filename, file):
+        try:
+            output = json.loads("".join(output))
+        except ValueError:
+            # This implies that the result is empty and there
+            # are no errors.
+            return
+
+        for issue in output:
+            yield Result.from_values(
+                origin=self,
+                message=issue["failure"],
+                file=issue["name"],
+                line=int(issue["startPosition"]["line"])+1,
+                column=issue["startPosition"]["character"],
+                end_column=issue["endPosition"]["character"])
+
+    def run(self,
+            filename,
+            file,
+            tslint_config: str="",
```

Comment:
Can you use the `path` in Settings ? This is something we're trying to get people to use from now on.